app/isomorphic/components/molecules/SocialLogin/index.js

```diff
@@ -1,102 +1,94 @@
-  // const [error, setError] = useState("");
-  // const { appKeys } = IntegrationData();
-  // const facebookKey = get(appKeys, ["facebookAppkey"], "");
-  // const googleClientId = get(appKeys, ["googleClientId"], "");
-  // const twitterKey = get(appKeys, ["twitterAppkey"], "");
+  const [error, setError] = useState("");
+
+  const socialLogin = (e, login) => {
+    e.preventDefault();
+
+    login()
+      .then(() => {
+        checkForMemberUpdated();
+        checkForMemberUpdated().then(res => {
+          console.log("successfully login");
+        });
+      })
+      .catch(error => {
+        console.log("error", error);
+        if (error === "NO_EMAIL") {
+          setError("The account you are using does not have an email id. Please try with another account.");
+        } else if (error === "NOT_LOADED") {
+          setError("");
+        } else if (error === "NOT_GRANTED") {
+          setError("There seems to be an error with social logins. Please do a manual email/password login.");
+        } else {
+          setError("Oops! Something went wrong. Please try again later.");
+        }
+      }); // Can also make an API call to /api/v1/members/me
+  };
 
-  // const socialLogin = (e, login) => {
-  //   e.preventDefault();
+  const googleOnClick = (e, serverSideLoginPath) => {
+    window.location.href = serverSideLoginPath;
+  };
 
-  //   login()
-  //     .then(() => {
-  //       checkForMemberUpdated();
-  //       checkForMemberUpdated().then(res => {
-  //         console.log("successfully login");
-  //       });
-  //     })
-  //     .catch(error => {
-  //       console.log("error", error);
-  //       if (error === "NO_EMAIL") {
-  //         setError("The account you are using does not have an email id. Please try with another account.");
-  //       } else if (error === "NOT_LOADED") {
-  //         setError("");
-  //       } else if (error === "NOT_GRANTED") {
-  //         setError("There seems to be an error with social logins. Please do a manual email/password login.");
-  //       } else {
-  //         setError("Oops! Something went wrong. Please try again later.");
-  //       }
-  //     }); // Can also make an API call to /api/v1/members/me
-  // };
+  const FaceBookLogin = () => {
+    const { login, serverSideLoginPath } = withFacebookLogin("248865019954260", "email", true);
+    return (
+      <Button color="#3b5998" flat href={serverSideLoginPath} onClick={e => socialLogin(e, login)} socialButton>
+        <span styleName="icon">
+          <FbIcon color="#3b5998" width={9} height={15} />
+        </span>{" "}
+        Facebook
+      </Button>
+    );
+  };
 
-  // const googleOnClick = (e, serverSideLoginPath) => {
-  //   window.location.href = serverSideLoginPath;
-  // };
+  const GoogleLogin = () => {
+    const { serverSideLoginPath } = withGoogleLogin(
+      "163120650123-m2rj93thcgkfs7js80cop6frppemfo1c.apps.googleusercontent.com",
```

Comment:
can you remove this and take it from config

Reply:
sure